packages/ui/src/components/ChangeKafkaOwner/ChangeKafkaOwner.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Alert,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  Form,
+  FormGroup,
+  Modal,
+} from "@patternfly/react-core";
+import { FunctionComponent, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { SelectOwner } from "./SelectOwner";
+import { OwnerAccount } from "./types";
+
+export type ChangeKafkaOwnerProps = {
+  isModalOpen: boolean;
+  currentOwner: string;
+  accounts: OwnerAccount[];
+  selectedAccount: string | undefined;
```

Comment:
This state shouldn't have been lifted up. The consumer of this component doesn't care if the user is fiddling with the options in the select; it just wants to know if the user did click confirm or not.